packages/@react-spectrum/actionbar/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@react-spectrum/actionbar",
-  "version": "3.0.0-alpha.3",
+  "version": "3.0.0-alpha.4",
```

Comment:
this will get taken care of when we do release

Reply:
@snowystinger  do you need me to undo this or can i merge it in as is?

Reply:
undo please